### ISSUE TITLE
Config should be in a dotfile

### ISSUE THREAD

**Issue by jayvdb** [jlaska/pytest-github#5]
It isnt a good idea to pit tokens in a project directory, and in this case the config file would be identical for all projects. So by default it should be stored in a file like ~/.github.yml

**Comment by jlaska** [comment]
@jayvdb fair point.  In the meantime, you're welcome to override the location of the config file (we actually do this locally) by passing the `--github-cfg` parameter.
